src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -264,6 +264,8 @@
       <EmscriptenEnvVars Include="PYTHONPATH=$(EmscriptenPythonToolsPath)" Condition="'$(OS)' == 'Windows_NT'" />
       <EmscriptenEnvVars Include="PYTHONHOME=" Condition="'$(OS)' == 'Windows_NT'" />
       <EmscriptenEnvVars Include="EM_CACHE=$(WasmCachePath)" Condition="'$(WasmCachePath)' != ''" />
+      <EmscriptenEnvVars Include="MonoWasmLegacyJsInterop=$(MonoWasmLegacyJsInterop)"/>
+      <EmscriptenEnvVars Include="MonoWasmThreads=$(MonoWasmThreads)"/>
```

Comment:
Should these be set only when they have value?

Reply:
They must have value even if we just default it to `false` or `true`.

The env variable is the way how to evaluate it in `dotnet.es6.lib.js` while linking.
But when this code would be running on user dev machine, I don't know how to get the value which was previously used to compile this runtime flavor in our CI. 

There was `USE_PTHREADS` which is pre-processor directive, but it works only for existing emcc settings. Whatever could be passed with `-s XXX`. Benefit is, that `emcc-link.rsp` would have that value stored for the runtime flavor.

But here I want to add something which emcc doesn't know about and so I went with env variables. But those are not in `emcc-link.rsp`. As I write this comment now I rememebred that there is `emcc-props.json` which could carry that value for me. That would work.

But is that wasted effort if we go for 2) as next step ?